packages/lesswrong/server/resolvers/jargonResolvers/exampleJargonPost.ts

```diff
@@ -0,0 +1,1852 @@
+<p>Once a robot can do everything an IQ 80 human can do, only better and cheaper, there will be no reason to employ IQ 80 humans. Once a robot can do everything an IQ 120 human can do, only better and cheaper, there will be no reason to employ IQ 120 humans. Once a robot can do everything an IQ 180 human can do, only better and cheaper, there will be no reason to employ humans at all, in the unlikely scenario that there are any left by that point.</p>
+<p>In the earlier stages of the process, capitalism becomes more and more uncoupled from its previous job as an optimizer for human values. Now most humans are totally locked out of the group whose values capitalism optimizes for. They have no value to contribute as workers &#8211; and since in the absence of a spectacular social safety net it&#8217;s unclear how they would have much money &#8211; they have no value as customers either. Capitalism has passed them by. As the segment of humans who can be outcompeted by robots increases, capitalism passes by more and more people until eventually it locks out the human race entirely, once again in the vanishingly unlikely scenario that we are still around.</p>
+<p>(there are some scenarios in which a few capitalists who own the robots may benefit here, but in either case the vast majority are out of luck)</p>
+<p>Democracy is less obviously vulnerable, but it might be worth going back to Bostrom&#8217;s paragraph about the Quiverfull movement. These are some really religious Christians who think that God wants them to have as many kids as possible, and who can end up with families of ten or more. Their <A HREF="http://www.patheos.com/blogs/lovejoyfeminism/2012/01/quiverfull-outbreeding-the-world.html">articles explictly calculate</A> that if they start at two percent of the population, but have on average eight children per generation when everyone else on average only has two, within three generations they&#8217;ll make up half the population.</p>
+<p>It&#8217;s a clever strategy, but I can think of one thing that will save us: judging by how many ex-Quiverfull blogs I found when searching for those statistics, their retention rates even within a single generation are pretty grim. Their article admits that 80% of very religious children leave the church as adults (although of course they expect their own movement to do better). And this is not a symmetrical process &#8211; 80% of children who grow up in atheist families aren&#8217;t becoming Quiverfull.</p>
+<p>It looks a lot like even though they are outbreeding us, we are outmeme-ing them, and that gives us a decisive advantage.</p>
+<p>But we should also be kind of scared of this process. Memes optimize for making people want to accept them and pass them on &#8211; so like capitalism and democracy, they&#8217;re optimizing for a <i>proxy</i> of making us happy, but that proxy can easily get uncoupled from the original goal.</p>
+</div>`
+
+export const exampleJargonGlossary2 = `[
```

Comment:
This feels like it can be an actual (well-typed) object instead of a string, and if you don't want to pay the cost of serializing it each time it's used in a query just do it once in the same file.  But the source should be an actual object.